docker-compose.yml

```diff
@@ -1,31 +1,54 @@
 version: "3.7"
 services:
   neo4j:
-    image: neo4j:3.5
+    image: neo4j:4.4.5-community
     restart: unless-stopped
     ports:
       - 7474:7474
       - 7687:7687
+    # expose:
+    #   - 7687
     volumes:
       - ./.compose/neo4j/conf:/conf
       - ./.compose/neo4j/data:/data
       - ./.compose/neo4j/import:/import
       - ./.compose/neo4j/logs:/logs
       - ./.compose/neo4j/plugins:/plugins
     environment:
-      # Raise memory limits
+      # Raise memory limits:
       - NEO4J_dbms_memory_pagecache_size=1G
       - NEO4J_dbms.memory.heap.initial_size=1G
       - NEO4J_dbms_memory_heap_max__size=1G
+      # Auth:
+      - NEO4J_AUTH=none
+      # Add APOC and GDS:
+      - apoc.export.file.enabled=true
+      - apoc.import.file.enabled=true
+      - apoc.import.file.use_neo4j_config=true
+      - NEO4JLABS_PLUGINS=["graph-data-science", "apoc"]
+      - NEO4J_dbms_security_procedures_whitelist=gds.*, apoc.*
+      - NEO4J_dbms_security_procedures_unrestricted=gds.*, apoc.*
+      # Networking:
+      - dbms.connector.bolt.listen_address=0.0.0.0:7687
+
     healthcheck:
         test: ["CMD", "curl", "-f", "http://localhost:7474"]
-        interval: 30s
+        interval: 10s
         timeout: 10s
-        retries: 5
+        retries: 10
   cartography:
-    image: lyft/cartography
-    user: cartography
+    build:
+      context: .
+      dockerfile: dist.Dockerfile
```

Comment:
Downside here is that it requires people to build the image to use it, right?

Reply:
Also, it should be possible to add this build info while also keeping the image/user settings, right?

Reply:
I think yes, it worked for me, but let's check it out.

Reply:
@ryan-lane what would you like to change? We need Neo and your functionality to work together. Please explain.

Reply:
Both the user and image should be left here. The user makes the container run as non-root, which is more secure. The image definition let's folks pull the image without needing to build it.

I'm not sure what this has to do with cartography and neo4j compatibility?

Reply:
@ryan-lane Sorry didn't get your point. Please give me an example of what do you mean and how would you like me to change it and why. I think I lost you a bit in this case.

Reply:
This is a working docker-compose setup, if someone wants to change anything, let them change or tell me exactly what should I change in order to finish this PR. Maybe we can add some explanations to the docs? Please advise how would you like to proceed.

Reply:
@achantavy